### DISCUSSION TITLE
Would it possible to exclude a specific show from syncing?

### DISCUSSION THREAD

**Discussion by PaxVobiscuit** [Taxel/PlexTraktSync#645] [Ideas]
I realize it is possible to exclude entire libraries, but would it be possible to add the ability to exclude specific folders, e.g. something  I have in TV shows but know doesn't exist in Trakt?

**Comment by simonc56** [comment]
Why don't you [add](https://www.themoviedb.org/tv/new) this show in [TMDB](https://blog.trakt.tv/tmdb-transition-ef3d19a5cf24) then ? What's the name of the show ?

The exclude logic assumes user organize its Plex libraries with identical types of video together in the same library.

**Reply by PaxVobiscuit** [reply]
> Why don't you [add](https://www.themoviedb.org/tv/new) this show in [TMDB](https://blog.trakt.tv/tmdb-transition-ef3d19a5cf24) then ? What's the name of the show ?
> 
> The exclude logic assumes user organize its Plex libraries with identical types of video together in the same library.

It's not really a "show" in the traditional sense, it is a series of videos my kids put together that I have on our Plex as a TV show.  When I run the tool it throws the "Not found in Trakt" message, which makes my "logs must be clean" brain unhappy.

**Reply by glensc** [reply]
Putting them to different Library and then excluding the library is a valid point.

But sometimes it's unsolvable because the data in trakt.tv does not match the naming of imdb. I know that American Dad, Beavis and Butthead, Futurama, Chilling Adventures of Sabrina, etc suffer this inconsistency.

- 7 episodes: https://trakt.tv/shows/american-dad/seasons/1
- 16 episodes: https://trakt.tv/shows/american-dad/seasons/2
- 23 episodes: https://www.imdb.com/title/tt0397306/episodes?season=1
- 19 episodes: https://www.imdb.com/title/tt0397306/episodes?season=2

basically trakt s01+s02 = imdb s01 (click to open replies): 
- https://trakt.tv/shows/american-dad/seasons/2/comments

**Comment by glensc** [comment]
Feature requests need to be submitted as issues for them to be developed.

Discussions are user-to-user (community) communication.
